package.json

```diff
@@ -103,6 +104,7 @@
     "react-plotly.js": "^2.6.0",
     "react-select": "^5.7.3",
     "react-syntax-highlighter": "^15.6.1",
+    "react-table": "^7.8.0",
```

Comment:
It seems that this is version 7 of `react-tool` and you already imported version 8 in `@tanstack/react-tool`.
I think we can delete this line.